src/nwb_datajoint/common/common_curation.py

```diff
@@ -100,50 +95,51 @@ def make(self, key):
         AnalysisNwbfile().add(key['nwb_file_name'], key['analysis_file_name'])       
 
-        # add sorting to Sortings
-        Sortings.insert1({'nwb_file_name': key['nwb_file_name'],
-                          'recording_id': key['recording_id'],
-                          'sorting_id': key['sorting_id'],
-                          'sorting_path': key['sorting_path'],
-                          'parent_sorting_id': key['parent_sorting_id']}, skip_duplicates=True)
+        # add sorting to Sorting
+        sorting_key = (SpikeSorting & key).proj().fetch1()
+        sorting_key['sorting_id'] = 'S_'+str(uuid.uuid4())[:8]
+        sorting_key['sorting_path'] = key['sorting_path']
+        sorting_key['parent_sorting_id'] = key['sorting_id']
+
+        SortingID.insert1(sorting_key, skip_duplicates=True)
         
         self.insert1(key)
    
     def update_manual_curation(self, key: dict):
         """Based on information in key, loads the curated sorting from sortingview,
-        saves it, and inserts it to Sortings
+        saves it, and inserts it to Sorting
         
         Assumes that the workspace corresponding to the recording and (original) sorting exists
 
         Parameters
         ----------
         key : dict
-            primary key of AutomaticCurationSorting
+            primary key of AutomaticCuration
         """
         workspace_uri = (SortingviewWorkspace & key).fetch1('workspace_uri')
         workspace = sv.load_workspace(workspace_uri=workspace_uri)
         
-        sortingview_sorting_id = (SortingviewWorkspace.Sortings & {'recording_id': key['recording_id'],
-                                                                    'sorting_id': key['sorting_id']}).fetch1('sortingview_sorting_id')
+        sortingview_sorting_id = (SortingviewWorkspaceSorting & key).fetch1('sortingview_sorting_id')
         manually_curated_sorting = workspace.get_curated_sorting_extractor(sorting_id=sortingview_sorting_id)
         sorting = si.create_sorting_from_old_extractor(manually_curated_sorting)
         
-        new_sorting_id = 'S_'+str(uuid.uuid4())[:8]
-        
         sorting_name = key['sorting_id']
         sorting_path = str(Path(os.getenv('NWB_DATAJOINT_SORTING_DIR')) / Path(sorting_name))
         sorting = sorting.save(folder=sorting_path)
         
-        Sortings.insert1({'recording_id': key['recording_id'],
-                          'sorting_id': new_sorting_id,
-                          'sorting_path': sorting_path,
-                          'parent_sorting_id': key['sorting_id']}, skip_duplicates=True)
+        # add sorting to Sortings
+        sorting_key = (SpikeSorting & key).proj().fetch1()
+        sorting_key['sorting_id'] = 'S_'+str(uuid.uuid4())[:8]
+        sorting_key['sorting_path'] = sorting_path
+        sorting_key['parent_sorting_id'] = key['sorting_id']
+        
+        SortingID.insert1(sorting_key, skip_duplicates=True)
```

Comment:
Again, how is the user going to access this curated sorting? The sorting_id is random. And there's no information about where the sorting came from (apart from the parent_sorting_id)